detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -106,7 +121,31 @@ public Container GetContainer()
                         try
                         {
                             string projectRelativePath = project.Path;
-                            string projectPath = PathUtil.Combine(solutionDirectory, projectRelativePath);
+                            string projectPath = null;
+                            Boolean directoryPackagesExists = false;
+                            if (projectRelativePath.Contains("Directory.Packages.props"))
+                            {
+                                string parentPath = solutionDirectory;
+                                bool fileNotFound = true;
+                                while (fileNotFound)
+                                {
+                                    parentPath =
+                                        parentPath.Substring(0, parentPath.LastIndexOf("/"));
+                                    string checkFile = Path.Combine(parentPath, projectRelativePath);
+                                    if (parentPath.Equals("/"))
+                                    {
+                                        Console.WriteLine("The Path provided in the sln file is wrong, will skip parsing over this file");
```

Comment:
The NuGet inspector in general seems to have inconsistent error handling. So, I wonder if here we should be giving up on parsing if we identified that something that is being included does not exist.